skaffold-operator.yaml

```diff
@@ -0,0 +1,115 @@
+apiVersion: skaffold/v4beta12
```

Comment:
You could also adapt https://github.com/gardener/gardener-extension-registry-cache/blob/ea2084a08c0a61511e901ea8cc0c8f94e3b7a6e2/hack/check-skaffold-deps.sh#L21-L22 to check the newly added `skaffold-operator.yaml` file.

Otherwise, `make check` won't fail on outdated dependencies in the `skaffold-operator.yaml` file.